### PR TITLE
Update special-fives.mdx

### PR DESCRIPTION
**Pull request by J-Fricke**
Add clarity to light-pink-fives convention

### CODE REVIEW

**Comment by Zamiell** [comment]
i dont understand the example you added, is it correct?

**Comment by J-Fricke** [comment]
> i dont understand the example you added, is it correct?

That's what people are saying, you can clue any rank to make a ligh-pink-five play. I'm simply clarifying what people seem to want with this convention. I don't agree with it, but I'm one person vs the many.

**Comment by J-Fricke** [comment]
> Why would bob mark the card as red 3, that doesn't make any sense

Agreed, updated to remove that part.

**Comment by Zamiell** [comment]
the example isn't accurate because it only applies to light pink 5s

**Comment by Zamiell** [comment]
can you change it to light pink instead of blue?

**Comment by J-Fricke** [comment]
> can you change it to light pink instead of blue?

The example is for the variant Light Pink Fives. All 5s are light pink here. Blue 5 is just the example 5 used as it's the suit used in the above example. It could be any color ultimately.

**Comment by Zamiell** [comment]
oh sorry i forgot the context

File: docs/variant-specific/special-fives.mdx [hunk 1]
```diff
@@ -102,10 +102,16 @@ These conventions apply to the "Special-Fives" variants (e.g. "Pink-Fives", "Mud
...
+  - The consequence of this is that when the light-pink 4 is down, _Self Finesses_ will no longer work.
 - For example, in a 3-player game:
   - Red 2 and blue 4 are played on the stacks.
   - Alice clues number 3 to Bob, touching a single card on slot 1.
   - Bob knows that this must be a _Play Clue_. Bob marks the card as either the red 3 or the blue 5.
+- For example, in a 3 player game:
+  - Red 1 and blue 4 are played on the stacks.
+  - Alice clues number 3 to Bob, touching a single card on slot 1.
+  - Bob knows that this must be a _Play Clue_. Bob marks the card blue 5.
+  - Since _Occam's Razor_ would say play least number of blind cards, Bob plays the "3" as blue 5 since _Self Finesse_ is disabled when 4s are played.
```
Comment (newduke) [inline docs/variant-specific/special-fives.mdx:115]: This example supersedes the former. Just replace it.

File: docs/variant-specific/special-fives.mdx [hunk 2]
```diff
@@ -102,10 +102,16 @@ These conventions apply to the "Special-Fives" variants (e.g. "Pink-Fives", "Mud
 
 - _Pink Promise_ does not apply to rank clues that could represent a playable light-pink-five. (This is different from how _Pink Promise_ works in a pink-fives variant.)
 - Thus, when players receive a rank _Play Clue_, they have to account for the being both the true rank and any playable light-pink-fives.
```
Comment (newduke) [inline docs/variant-specific/special-fives.mdx:104]: "account for the being both the true rank"

the what?

File: docs/variant-specific/special-fives.mdx [hunk 3]
```diff
@@ -102,10 +102,16 @@ These conventions apply to the "Special-Fives" variants (e.g. "Pink-Fives", "Mud
...
+  - The consequence of this is that when the light-pink 4 is down, _Self Finesses_ will no longer work.
```
Comment (newduke) [inline docs/variant-specific/special-fives.mdx:105]: This is about special 5s, not special 5s + light-pink. Please clarify that *any* 4 being down *might* prevent a self finesse where exactly occam's razor has priority over, say, another player blind playing.

**Review by newduke — COMMENTED** [review]
Thanks!

File: docs/variant-specific/special-fives.mdx (hunk 3 above)
Comment (J-Fricke) [reply docs/variant-specific/special-fives.mdx]: @newduke New commit should include the above changes.

**Comment by Zamiell** [comment]
i edited the pr a bit, you can review it

**Pull request merged**